src/Sylius/Bundle/ImportExportBundle/Behat/ImportExportContext.php

```diff
@@ -0,0 +1,637 @@
+     * @param bool|true $flush
+     *
+     * @return ImportProfileInterface
+     */
+    private function thereIsImportProfile(
+        $name,
+        $description,
+        $code,
+        $reader,
+        $readerConfiguration,
+        $writer,
+        $writerConfiguration,
+        $flush = true
+    ) {
+        $repository = $this->getRepository('import_profile');
+
+        return $this->createProfileInDatabase($repository, $description, $code, $reader, $readerConfiguration, $writer, $writerConfiguration, $name, $flush);
+    }
+
+    /**
+     * @param string    $name
+     * @param string    $description
+     * @param string    $code
+     * @param string    $reader
+     * @param string    $readerConfiguration
+     * @param string    $writer
+     * @param string    $writerConfiguration
+     * @param bool|true $flush
+     *
+     * @return ExportProfileInterface
+     */
+    private function thereIsExportProfile(
+        $name,
+        $description,
+        $code,
+        $reader,
+        $readerConfiguration,
+        $writer,
+        $writerConfiguration,
+        $flush = true
+    ) {
+        $repository = $this->getRepository('export_profile');
+
+        return $this->createProfileInDatabase($repository, $description, $code, $reader, $readerConfiguration, $writer, $writerConfiguration, $name, $flush);
+    }
+
+    /**
+     * @param string    $status
+     * @param string    $startTime
+     * @param string    $endTime
+     * @param string    $createdAt
+     * @param string    $updatedAt
+     * @param string    $exportProfileCode
+     * @param bool|true $flush
+     *
+     * @return ExportJobInterface
+     */
+    private function thereIsExportJob($status, $startTime, $endTime, $createdAt, $updatedAt, $exportProfileCode, $flush = true)
+    {
+        $repository = $this->getRepository('export_job');
+        $exportJob = $repository->createNew();
+        $exportJob->setStatus($status);
+        $exportJob->setStartTime(new \DateTime($startTime));
+        $exportJob->setEndTime(new \DateTime($endTime));
+        $exportJob->setCreatedAt(new \DateTime($createdAt));
+        $exportJob->setUpdatedAt(new \DateTime($updatedAt));
+
+        $exportProfile = $this->getRepository('export_profile')->findOneByCode($exportProfileCode);
+        $exportJob->setProfile($exportProfile);
+
+        $manager = $this->getEntityManager();
+        $manager->persist($exportJob);
+
+        if ($flush) {
+            $manager->flush();
+        }
+
+        return $exportJob;
+    }
+
+    /**
+     * @param string    $status
+     * @param string    $startTime
+     * @param string    $endTime
+     * @param string    $createdAt
+     * @param string    $updatedAt
+     * @param string    $importProfileCode
+     * @param bool|true $flush
+     *
+     * @return ImportJobInterface
+     */
+    private function thereIsImportJob($status, $startTime, $endTime, $createdAt, $updatedAt, $importProfileCode, $flush = true)
```

Comment:
It's almost the same as `thereIsExportJob`, only thing that changes is `export` turns into `import`. I think it can be extracted to one method.